mirrord/cli/src/main.rs

```diff
@@ -102,6 +103,85 @@ where
     // Put original executable in argv[0] even if actually running patched version.
     binary_args.insert(0, args.binary.clone());
 
+    // Print config details for the user
+    sub_progress.info(
```

Comment:
If you use a separate `Progress` subtask for printing this out (`Progress::subtask`), it will be easier to find this info in the plugins (you'll get a JSON message like `{"type": "new_task", "name": "you_choose"}`)

Reply:
Also, you could put this in a separate function ^^